testdata/gate.yml

```diff
@@ -64,5 +64,7 @@ google:
     audience: projects/0/apps/0
     iapVerifyKeyUrl: https://my-url.test/
 services:
+  deck:
+    baseUrl: https://spinnaker.test:9000
```

Comment:
My only question was going to be, where's the halconfig.yml update?! But just checked and you already did that in a prior PR! 🎉 🎉